ergo/distributions/conditions.py

```diff
@@ -216,12 +209,12 @@ def loss(self, dist):
         return self.weight * total_loss
 
     def get_normalized(self, scale_min: float, scale_max: float):
-        scale_range = scale_max - scale_min
-        return self.__class__(self.weight, (self.loc_mean - scale_min) / scale_range)
+        scale = Scale(scale_min, scale_max)
+        return self.__class__(self.weight, scale.normalize_point(self.loc_mean))
 
-    def get_denormalized(self, scale_min, scale_max):
-        scale_range = scale_max - scale_min
-        return self.__class__(self.weight, (self.loc_mean * scale_range) + scale_min)
+    def get_denormalized(self, scale_min: float, scale_max: float):
+        scale = Scale(scale_min, scale_max)
```

Comment:
I wonder if the caller should pass a scale instead of a min/max.

Reply:
That'd be reasonable. I guess part of what I was thinking is that these functions (well maybe not these ones, but the LogisticMixture `normalize` function) might be called by application code (e.g. Elicit code), and it seems a bit weird to have to pass in a random Scale object in that case

Reply:
TBC I'm on the fence here and will change if you think it's better to change

Reply:
Seems better to me to change, especially if it helps with log scales (not sure it does).

Reply:
Cool, I'll change it